### PR TITLE
MCKIN-8072 Highlight target zone until popup is closed

### PR DESCRIPTION
**Pull request by jawad-khan**
Highlight target zone until popup is closed.

### CODE REVIEW

**Comment by Agrendalath** [comment]
Hey, @jawad-khan, I've tested it and I've noticed two cases when it does not behave as described:
1. When the popup is displayed, you can click `Reset` button. If you do it, the outline will be visible until you close the next popup:
reset
2. If you enable `Display zone borders on the image` (in component's `Zone borders` settings), the outline will not appear with the popup:
zone_borders

**Comment by jawad-khan** [comment]
@Agrendalath Can you please look it again?

File: drag_and_drop_v2/public/css/drag_and_drop.css [hunk 1]
```diff
@@ -385,8 +385,14 @@
...
 
+.xblock--drag-and-drop .zone-with-borders {
+    border: 1px dotted #565656;
+}
+
 /*To highlight ZONES while user is dragging an item*/
-.xblock--drag-and-drop .drag-container.dragging .zone{
+.xblock--drag-and-drop .drag-container.dragging .zone,
+.green-zone,
+.zone-with-borders.green-zone{
```
Comment (Agrendalath) [inline drag_and_drop_v2/public/css/drag_and_drop.css]: Please add the space before the brace.

File: drag_and_drop_v2/public/js/drag_and_drop.js [hunk 2]
```diff
@@ -1871,6 +1871,11 @@ function DragAndDropBlock(runtime, element, configuration) {
                 if (state.feedback && state.feedback.length > 0) {
                     // Move focus the the close button of the feedback popup.
                     focusFeedbackPopup('.item-feedback-popup', '.close-feedback-popup-button:visible');
+                    if (configuration.mode == DragAndDropBlock.STANDARD_MODE){
+                        var zone_selector = ".zone[data-uid='" + zone + "']";
+                        $root.find(zone_selector).addClass("green-zone");
+
```
Comment (Agrendalath) [inline drag_and_drop_v2/public/js/drag_and_drop.js]: Please get rid of the unnecessary newline.

File: drag_and_drop_v2/public/js/drag_and_drop.js [hunk 3]
```diff
@@ -1871,6 +1871,11 @@ function DragAndDropBlock(runtime, element, configuration) {
                 if (state.feedback && state.feedback.length > 0) {
                     // Move focus the the close button of the feedback popup.
                     focusFeedbackPopup('.item-feedback-popup', '.close-feedback-popup-button:visible');
+                    if (configuration.mode == DragAndDropBlock.STANDARD_MODE){
```
Comment (Agrendalath) [inline drag_and_drop_v2/public/js/drag_and_drop.js]: Please add the comment for this block and put the space before the brace.

File: drag_and_drop_v2/public/js/drag_and_drop.js [hunk 4]
```diff
@@ -1908,6 +1913,7 @@ function DragAndDropBlock(runtime, element, configuration) {
     var closePopup = function(manually_closed) {
         // do not apply state here - callers are responsible to call it when other appropriate state changes are applied
         if ($root.find(Selector.popup_box).is(":visible")) {
+            $root.find(".zone").removeClass('green-zone')
```
Comment (Agrendalath) [inline drag_and_drop_v2/public/js/drag_and_drop.js]: Please add the semicolon.

File: drag_and_drop_v2/public/js/drag_and_drop.js [hunk 5]
```diff
@@ -1939,6 +1945,8 @@ function DragAndDropBlock(runtime, element, configuration) {
             state = data;
             applyState();
             focusFirstDraggable();
+            $root.find(".zone").removeClass('green-zone')
+
```
Comment (Agrendalath) [inline drag_and_drop_v2/public/js/drag_and_drop.js]: Please add the semicolon and remove the unnecessary newline.

**Comment by jawad-khan** [comment]
@Agrendalath can you please look it again. Also if you everything is fine, merge with squashing commit option.

**Pull request merged**